approximate_baseline/approximateBandpasses.py

```diff
@@ -0,0 +1,35 @@
+import os
+import numpy as np
+from lsst.utils import getPackageDir
```

Comment:
Actually, I can set this up using `__file__`. And I can prevent the writing to the real `baseline` directory. I also chaged it so that it can be run from other directories, and it will write to the correct location.